pandas/tests/copy_view/test_methods.py

```diff
@@ -72,7 +72,7 @@ def test_copy_shallow(using_copy_on_write):
         lambda df, copy: df.rename_axis(columns="test", copy=copy),
         lambda df, copy: df.astype({"b": "int64"}, copy=copy),
         # lambda df, copy: df.swaplevel(0, 0, copy=copy),
-        lambda df, copy: df.swapaxes(0, 0, copy=copy),
+        # lambda df, copy: df.swapaxes(0, 0, copy=copy),
```

Comment:
not sure some like this, should be comment or deleted

Reply:
This should be kept